learningresources/tests/base.py

```diff
@@ -155,5 +177,5 @@ def get_course_single_tarball(self):
         Returns:
             path (unicode): absolute path to tarball.
         """
-        path = join(abspath(dirname(__file__)), "testdata", "courses")
-        return self.copy_file(join(path, "single.tgz"))
+        path = join(abspath(dirname(__file__)), "testdata", "courses", "toy")
+        return self._make_archive(path, False, '.tgz')
```

Comment:
I need to read the code more closely, but here the extension is ``tgz`` and yet in ``make_archive()`` the default is ``tar.gz``.  Should these both be the same, or is there something that make these two unrelated?